src/scp/LocalNode.cpp

```diff
@@ -372,17 +373,27 @@ LocalNode::findClosestVBlocking(SCPQuorumSet const& qset,
 
 Json::Value
 LocalNode::toJson(SCPQuorumSet const& qSet, bool fullKeys) const
+{
+    using std::placeholders::_1;
```

Comment:
does not seem to be used

Reply:
this was deleted in a later commit, I'll clean this one up